app-emulation/docker/docker-9999.ebuild

```diff
@@ -198,13 +203,21 @@ pkg_setup() {
 	enewgroup docker
 }
 
+src_prepare() {
```

Comment:
Can't this just have a `PATCHES` array instead of adding a `src_prepare` function?